docs/authoring.md

```diff
@@ -115,9 +90,68 @@ Similarly, any other dependencies, e.g. `Microsoft.WinUI`, will need to be inclu
 </files>
 ```
 
-Your component can then be added as a PackageReference to any consumer. 
+Your component can then be used in consuming apps by a `PackageReference`.
+
+## Consuming C#/WinRT Components
+
+This section describes the steps needed to consume a C#/WinRT component from the following kinds of applications:
+
+- [C++/WinRT desktop applications](#Consuming-from-C++/WinRT)
+- [C# .NET 5+ desktop applications](#Consuming-from-C#-applications)
+- [Out of process components](#Consuming-an-out-of-process-component)
+- [Packaged applications](#Consuming-from-packaged-applications)
+
+### Consuming from C++/WinRT
+
+Consuming a C#/WinRT component from a C++/WinRT desktop application is supported by both package reference or project reference.
+
+- For package references, simply right-click on the native project node and click **Manage NuGet packages** to find and install the component package.
+
+- For project references, you also currently need a reference to the component's generated WinMD. The WinMD can be found in the output directory of the authored component's project. To add both the project and WinMD references, right-click on the native project node, and click **Add** -> **Reference**. Select the C#/WinRT component project under the **Projects** node and the generated WinMD file from the **Browse** node.
+
+For native consumption of C#/WinRT components, you also need to create a manifest file named `YourNativeApp.exe.manifest`. If your app is packaged with MSIX, then you don't need to include the manifest file, otherwise you need to include your activatable class registrations in the manifest file. We provide an [authoring sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/AuthoringDemo/CppConsoleApp) with an example manifest file. To create the manifest file:

```

Comment:
"If your app is packaged with MSIX, then you don't need to create a manifest file. In the case that you do make a manifest, you need to add activatable class registrations for the public types in your component."